packages/basic-crawler/src/internals/basic-crawler.ts

```diff
@@ -1414,6 +1453,22 @@ export class BasicCrawler<Context extends CrawlingContext = BasicCrawlingContext
 
         return request.headers?.Cookie || request.headers?.cookie || '';
     }
+
+    private _getRequestQueue() {
+        if (this.experiments.useRequestQueueV2) {
+            if (!this._experimentWarnings.useRequestQueueV2) {
+                this.log.warning([
+                    'The RequestQueue v2 is an experimental feature, and may have issues when used in a production environment.',
+                    'Please report any issues you encounter on GitHub: https://github.com/apify/crawlee',
+                ].join('\n'));
+                this._experimentWarnings.useRequestQueueV2 = true;
+            }
+
+            return RequestQueueV2.open();
+        }
+
+        return RequestQueue.open();
```

Comment:
those calls need to respect the current config, i just fixed that in master via https://github.com/apify/crawlee/commit/845141d921c10dd5fb121a499bb1b24f5eb3ff04

Reply:
Done